### DISCUSSION TITLE
Why is the apple_id specified in Appfile?

### DISCUSSION THREAD

**Discussion by swrobel** [fastlane/fastlane#17953] [Q&A]
The docs do not specify that the Appfile should be excluded from source control, but they contain a user-specific value, `apple_id`. Most of the other data in this file is shared between users (app identifiers, teams), however including the apple id means that the Appfile can't easily be kept in source control or there will be conflicts between users. What am I missing here?

**Comment by max-ott** [comment]
I recommend replacing the actual apple_id with an environment variable. Then you can reference the Apple ID and keep that in the appfile, without the need to run additional commands / parameters on the lanes. And each dev can have their own `.env` file containing private & sensitive information.
